packages/lesswrong/components/votes/lwReactions/AddInlineReactionButton.tsx

```diff
@@ -66,6 +66,11 @@ const AddInlineReactionButton = ({voteProps, classes, quote, commentItemRef}: {
     !disabled && setOpen(true)
   }
 
+  function handleToggleReaction (reaction: string, quote: string) {
```

Comment:
nit: not the function syntax we usually use

Reply:
It's about 1/3-2/3 between the `const name = () => {` style and the `function name () {` style; both are fine.